Ce glider

s-pearce opened the pull request:
@cdobs @cwingard is taking some time off, could you review this AM update and merge?  Thanks.

@rtravis-whoi you checked my AM updates last time I added a new G3 glider and I have done so again.  Could you review that I added new glider 917 in the right places again?  Thanks.

My updates did pass the unit tests.

-Stuart

rtravis-whoi commented on bulk/sensor_bulk_load-AssetRecord.csv:
```diff
@@ -2060,67 +2064,68 @@ CGINS-PARADK-20582,,Sensor,0,Photosynthetically Available Radiation: PARAD Serie
+CGINS-PARADM-50245,,Sensor,1,Photosynthetically Available Radiation for Glider: PARAD Series M,PARAD for Slocum Glider,Biospherical Instruments,QSP-2155,50245,1.2,,,,
+CGINS-PARADM-50252,,Sensor,1,Photosynthetically Available Radiation for Glider: PARAD Series M,PARAD for Slocum Glider,Biospherical Instruments,QSP-2155,50252,,,,,
+CGINS-PARADM-50277,,Sensor,1,Photosynthetically Available Radiation for Glider: PARAD Series M,PARAD for Slocum Glider,Biospherical Instruments,QSP-2155,50277,,,,,Woods Hole Oceanographic Institution
+CGINS-PARADM-50304,,Sensor,1,Photosynthetically Available Radiation for Glider: PARAD Series M,PARAD for Slocum Glider,Biospherical Instruments,QSP-2155,50304,,,,,
```
The cleanup looks ok - w/ removal of 'CGSN Glider Instrument Pool' & 'From Teledyne Webb Research's OOI glider instrument pool' from Comments column

s-pearce replied:
I removed those because TWR no longer operates an instrument pool between WHOI and OSU (where instruments might swap between MIOs).  We now keep our own set of instruments.

rtravis-whoi replied:
Sounds good thx.

snwhite replied:
That is fine.  We should add in the MIO for these when we can.

rtravis-whoi commented on bulk/sensor_bulk_load-AssetRecord.csv:
```diff
@@ -437,18 +437,19 @@ CGINS-ADCPAM-49697,N00330,Sensor,1,ADCP Velocity Profiler (short range) for Glid
 CGINS-ADCPAM-72777,,Sensor,1,ADCP Velocity Profiler (short range) for Glider: ADCPA Series M,ADCP for Slocum Glider,Teledyne RDI,Explorer DVL 600 kHz,672777,,,,,
 CGINS-ADCPAM-73107,,Sensor,1,ADCP Velocity Profiler (short range) for Glider: ADCPA Series M,ADCP for Slocum Glider,Teledyne RDI,Explorer DVL 600 kHz,673107,,,,,
 CGINS-ADCPAM-73245,,Sensor,1,ADCP Velocity Profiler (short range) for Glider: ADCPA Series M,ADCP for Slocum Glider,Teledyne RDI,Explorer DVL 600 kHz,673245,,,,,
+CGINS-ADCPAM-73246,,Sensor,1,ADCP Velocity Profiler (short range) for Glider: ADCPA Series M,ADCP for Slocum Glider,Teledyne RDI,Explorer DVL 600 kHz,673246,34.20,20210403,,,Oregon State University
```
looks good

rtravis-whoi commented on bulk/sensor_bulk_load-AssetRecord.csv:
```diff
@@ -614,87 +615,88 @@ CGINS-CTDBPP-50159,A01730,Sensor,0,CTD Pumped: CTDBP Series P,CTDBP SERIES P,Sea
+CGINS-CTDGVM-09453,,Sensor,1,CTD for Glider: CTDGV Series M,CTDGV for Slocum Glider,Sea-Bird Electronics,SBE Slocum Glider Payload CTD (GP-CTD),9453,,,,,Woods Hole Oceanographic Institution
 CGINS-CTDGVM-09658,,Sensor,1,CTD for Glider: CTDGV Series M,CTDGV for Slocum Glider,Sea-Bird Electronics,SBE Slocum Glider Payload CTD (GP-CTD),9658,,,,,
 CGINS-CTDGVM-09690,,Sensor,1,CTD for Glider: CTDGV Series M,CTDGV for Slocum Glider,Sea-Bird Electronics,SBE Slocum Glider Payload CTD (GP-CTD),9690,,,,,
+CGINS-CTDGVM-09692,,Sensor,1,CTD for Glider: CTDGV Series M,CTDGV for Slocum Glider,Sea-Bird Electronics,SBE Slocum Glider Payload CTD (GP-CTD),9692,1.3.1,20210403,,,Oregon State University
```
looks good

snwhite replied:
Looks fine.

rtravis-whoi commented on bulk/sensor_bulk_load-AssetRecord.csv:
```diff
@@ -1166,91 +1168,92 @@ CGINS-DOSTAL-02195,OL000042,Sensor,0,Dissolved Oxygen Stable Response: DOSTA Ser
+CGINS-DOSTAM-00605,,Sensor,1,Dissolved Oxygen Stable Response for Glider: DOSTA Series M,DOSTA for Slocum Glider,Aanderaa,Optode 4831,605,,,,,
+CGINS-DOSTAM-00662,,Sensor,1,Dissolved Oxygen Stable Response for Glider: DOSTA Series M,DOSTA for Slocum Glider,Aanderaa,Optode 4831,662,,,,,Woods Hole Oceanographic Institution
+CGINS-DOSTAM-00864,,Sensor,1,Dissolved Oxygen Stable Response for Glider: DOSTA Series M,DOSTA for Slocum Glider,Aanderaa,Optode 4831,864,,,,,Oregon State University
+CGINS-DOSTAM-00891,,Sensor,1,Dissolved Oxygen Stable Response for Glider: DOSTA Series M,DOSTA for Slocum Glider,Aanderaa,Optode 4831,891,5.1.1,20210403,,,Oregon State University
```
looks good

snwhite replied:
Looks fine.

rtravis-whoi commented on bulk/sensor_bulk_load-AssetRecord.csv:
```diff
@@ -1426,79 +1429,80 @@ CGINS-FLORTK-01537,,Sensor,0,3-Wavelength Fluorometer: FLORT Series K,,WET Labs,
+CGINS-FLORTM-04473,,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series M,FLORT for Coastal Slocum Glider,WET Labs,ECO Puck FLBBCD-SLK,4473,,,,,
+CGINS-FLORTM-05133,,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series M,FLORT for Coastal Slocum Glider,WET Labs,ECO Puck FLBBCD-SLK,5133,,,,,Woods Hole Oceanographic Institution
 CGINS-FLORTM-06168,,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series M,FLORT for Coastal Slocum Glider,WET Labs,ECO Puck FLBBCD-SLC,6168,,,,,
+CGINS-FLORTM-06744,,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series M,FLORT for Coastal Slocum Glider,WET Labs,ECO Puck FLBBCD-SLC,6744,4.07D,20210403,,,Oregon State University
```
looks good

snwhite replied:
Looks fine.

rtravis-whoi commented on bulk/sensor_bulk_load-AssetRecord.csv:
```diff
@@ -2060,67 +2064,68 @@ CGINS-PARADK-20582,,Sensor,0,Photosynthetically Available Radiation: PARAD Serie
-CGINS-PARADM-50333,,Sensor,1,Photosynthetically Available Radiation for Glider: PARAD Series M,PARAD for Slocum Glider,Biospherical Instruments,QSP-2155,50333,,,,CGSN Glider Instrument Pool,
+CGINS-PARADM-50324,,Sensor,1,Photosynthetically Available Radiation for Glider: PARAD Series M,PARAD for Slocum Glider,Biospherical Instruments,QSP-2155,50324,,,,,
+CGINS-PARADM-50333,,Sensor,1,Photosynthetically Available Radiation for Glider: PARAD Series M,PARAD for Slocum Glider,Biospherical Instruments,QSP-2155,50333,,,,,
+CGINS-PARADM-50336,,Sensor,1,Photosynthetically Available Radiation for Glider: PARAD Series M,PARAD for Slocum Glider,Biospherical Instruments,QSP-2155,50336,1.2,20210403,,,Oregon State University
```
looks good

snwhite replied:
Looks fine.

rtravis-whoi commented on calibration/ADCPAM/CGINS-ADCPAM-73246__20210304.csv:
```diff
@@ -0,0 +1,5 @@
+673246,CC_scale_factor1,0.61,
+673246,CC_scale_factor2,0.61,
+673246,CC_scale_factor3,0.61,
+673246,CC_scale_factor4,0.61,
```
looks correct, but no calibration/configuration file available on vault to verify date.

rtravis-whoi commented on calibration/CTDGVM/CGINS-CTDGVM-09692__20210203.csv:
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+9692,CC_calibration_date,1612310400000,sensor calibration date (milliseconds since 1970-01-01)
```
looks correct, but no calibration file available on vault/alfresco to verify cal date

rtravis-whoi replied:
confirmed - looks good

rtravis-whoi commented on calibration/DOSTAM/CGINS-DOSTAM-00891__20200614.csv:
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+891,CC_calibration_date,1592092800000,sensor calibration date (milliseconds since 1970-01-01)
```
looks correct, but no calibration file available on vault/alfresco to verify cal date

rtravis-whoi replied:
confirmed - looks good

rtravis-whoi commented on calibration/FLORTM/CGINS-FLORTM-06744__20210318.csv:
```diff
@@ -0,0 +1,5 @@
+6744,CC_angular_resolution,1.076,
+6744,CC_depolarization_ratio,0.039,
+6744,CC_measurement_wavelength,700,
+6744,CC_scattering_angle,124,
```
looks correct, but no calibration file available on vault/alfresco to verify cal date

rtravis-whoi replied:
confirmed - looks good

rtravis-whoi commented on calibration/PARADM/CGINS-PARADM-50336__20200817.csv:
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50336,CC_bsipar_par_scaling,1,
```
looks correct, but no calibration file available on vault/alfresco to verify cal date

s-pearce replied:
@rtravis-whoi I have now uploaded the calibration files to Alfresco (minus the ADCPAM docs since there is not a space for those in the usual calibration docs directories)

[Configuration](https://alfresco.oceanobservatories.org/alfresco/d/d/workspace/SpacesStore/685cd1eb-bd2c-419f-84f2-03b4c8956a37/8304-61003-917-00001_Configuration_Record.xlsx)

[PARAD](https://alfresco.oceanobservatories.org/alfresco/d/d/workspace/SpacesStore/4ca62f3b-04ae-4d3c-8443-59837f3ff432/PARAD-M_QSP2155_SN_50336_Calibration_2020-08-17.pdf)
[DOSTA](https://alfresco.oceanobservatories.org/alfresco/d/d/workspace/SpacesStore/516ade1a-7178-4c30-9594-2e21f89bcb48/DOSTA-M_Optode-4831_SN_891_Calibration_2020-06-14.pdf)
[FLORT](https://alfresco.oceanobservatories.org/alfresco/d/d/workspace/SpacesStore/52376cfe-49f5-4100-be35-7dcb0e048084/FLORT-M_FLBBCDSLC_SN_6744_Calibration_2021-03-18.pdf)
[CTDGV](https://alfresco.oceanobservatories.org/alfresco/d/d/workspace/SpacesStore/e569c30f-105f-428a-9634-2d549a73f8b5/CTDGV-M_SBE-Slocum_SN_9692_Calibration_2021-02-03.pdf)

rtravis-whoi replied:
confirmed - looks good

rtravis-whoi commented on cruise/CruiseInformation.csv:
```diff
@@ -256,5 +256,6 @@ EK20210714,Elakha,2021-07-14T15:12:00,2021-07-14T23:48:00,Endurance Glider opera
 EK20210715,Elakha,2021-07-15T15:30:00,2021-07-15T21:30:00,Endurance ce02shsp r00021 d00022 and O2 samples ce01 ce02
 SKQ202111S,Sikuliaq,2021-07-18T00:00:00,2021-08-02T23:59:00,Papa 8
 WJ20210721,Warren Jr.,2021-07-21T21:00:00,2021-07-23T09:00:00,Pioneer glider cp_379 recovery
+EK20210730,Elakha,2021-07-30T15:39:00,2021-07-30T18:37:00,Endurance glider deployment ce_917-00001
```
looks good (assuming the times are correct - no way to verify)

s-pearce replied:
I verify they are correct.  Excerpt from my cruise log (first time PDT second UTC):
08:39/15:39 left the dock
...
11:02/18:02 arrived back at the dock

rtravis-whoi replied:
Looks good - thx!

snwhite replied:
Looks fine.

rtravis-whoi commented on deployment/CE05MOAS-G0917_Deploy.csv:
```diff
@@ -0,0 +1,7 @@
+EK20210730,S.Pearce,,,CE05MOAS-G0917-02-FLORTM000,1,1,2021-07-30T16:45:00,,CGVCE-05MOAS-00000,CGVEH-GLDRCE-00917,CGINS-FLORTM-06744,44.646,-124.128,,200,200,On rechargeable lithium batteries,,
+EK20210730,S.Pearce,,,CE05MOAS-G0917-03-ADCPAM000,1,1,2021-07-30T16:45:00,,CGVCE-05MOAS-00000,CGVEH-GLDRCE-00917,CGINS-ADCPAM-73246,44.646,-124.128,,200,200,On rechargeable lithium batteries,,
+EK20210730,S.Pearce,,,CE05MOAS-G0917-04-DOSTAM000,1,1,2021-07-30T16:45:00,,CGVCE-05MOAS-00000,CGVEH-GLDRCE-00917,CGINS-DOSTAM-00891,44.646,-124.128,,200,200,On rechargeable lithium batteries,,
+EK20210730,S.Pearce,,,CE05MOAS-G0917-05-CTDGVM000,1,1,2021-07-30T16:45:00,,CGVCE-05MOAS-00000,CGVEH-GLDRCE-00917,CGINS-CTDGVM-09692,44.646,-124.128,,200,200,On rechargeable lithium batteries,,
```
These look ok, but no way to verify the sensor SNs - no FCP or sensor cal files in Vault.  Also checked RDB & Alfresco and didn't see any corresponding files.

s-pearce replied:
@rtravis-whoi I have now uploaded the calibration files to Alfresco (minus the ADCPAM docs since there is not a space for those in the usual calibration docs directories)

[Configuration](https://alfresco.oceanobservatories.org/alfresco/d/d/workspace/SpacesStore/685cd1eb-bd2c-419f-84f2-03b4c8956a37/8304-61003-917-00001_Configuration_Record.xlsx)

[PARAD](https://alfresco.oceanobservatories.org/alfresco/d/d/workspace/SpacesStore/4ca62f3b-04ae-4d3c-8443-59837f3ff432/PARAD-M_QSP2155_SN_50336_Calibration_2020-08-17.pdf)
[DOSTA](https://alfresco.oceanobservatories.org/alfresco/d/d/workspace/SpacesStore/516ade1a-7178-4c30-9594-2e21f89bcb48/DOSTA-M_Optode-4831_SN_891_Calibration_2020-06-14.pdf)
[FLORT](https://alfresco.oceanobservatories.org/alfresco/d/d/workspace/SpacesStore/52376cfe-49f5-4100-be35-7dcb0e048084/FLORT-M_FLBBCDSLC_SN_6744_Calibration_2021-03-18.pdf)
[CTDGV](https://alfresco.oceanobservatories.org/alfresco/d/d/workspace/SpacesStore/e569c30f-105f-428a-9634-2d549a73f8b5/CTDGV-M_SBE-Slocum_SN_9692_Calibration_2021-02-03.pdf)

rtravis-whoi replied:
Yah - I just realized that there isn't an ADCPA folder in Alfresco....not really sure why.  ADCPA folder has been added.

rtravis-whoi commented on bulk/eng_vocab.csv:
```diff
@@ -1,18 +1,18 @@
+Global Wire-Following Profiler Controller,McLane,MMP 210-3512
+Mobile Asset Controller for Coastal Glider,Teledyne Webb,G2 or G3 Slocum Gliders
+Mobile Asset Controller for Profiling Glider,Teledyne Webb,G2 or G3 Slocum Gliders
+Mobile Asset Controller for Open Ocean Glider,Teledyne Webb,G2 or G3 Slocum Gliders
```
addition of 'or G3' looks good

s-pearce replied:
Wasn't sure whether to add "or G3" or just remove the model numbers.

snwhite replied:
That seems fine.  We definitely have both, so adding is good.

rtravis-whoi commented on bulk/eng_bulk_load-AssetRecord.csv:
```diff
@@ -1,544 +1,545 @@
+CGCON-GLDRCE-00384,OL000370,Sensor,1,Mobile Asset Controller for Coastal Glider,GL384 Controller,Teledyne Webb,G2 Slocum Gliders,384,,,,,Oregon State University
+CGCON-GLDRCE-00386,OL000371,Sensor,1,Mobile Asset Controller for Coastal Glider,GL386 Controller,Teledyne Webb,G2 Slocum Gliders,386,,,,,Oregon State University
+CGCON-GLDRCE-00871,,Sensor,1,Mobile Asset Controller for Coastal Glider,G0871 Controller,Teledyne Webb,G3 Slocum Gliders,871,,,,,Oregon State University
+CGCON-GLDRCE-00917,,Sensor,1,Mobile Asset Controller for Coastal Glider,G0917 Controller,Teledyne Webb,G3 Slocum Gliders,917,,20210403,,,Oregon State University
```
addition for glider 917 looks good

snwhite replied:
This addition looks fine.

rtravis-whoi reviewed (changes requested):
One correction requested in node_bulk_load-AssetRecord.csv (see individual comment).  Also, not sure how to verify that the sensor SNs are correct for glider 917, or how to verify sensor calibration dates.  Cannot locate glider FCP or sensor cals for verification.

s-pearce commented on bulk/node_bulk_load-AssetRecord.csv:
```diff
@@ -1,167 +1,168 @@
+CGVEH-GLDRCE-00384,A00801,Node,1,Coastal Glider 384,COASTAL GLIDER 1000M,Teledyne Webb,CSG-1000,384,,20110628,186935,,Oregon State University
+CGVEH-GLDRCE-00386,A00473,Node,1,Coastal Glider 386,COASTAL GLIDER 1000M,Teledyne Webb,DG-1000,386,,20131011,186935,,Oregon State University
+CGVEH-GLDRCE-00871,,Node,1,Coastal Glider 871,COASTAL GLIDER 1000M,Teledyne Webb,GLD-0075_D,871,,20200813,,,Oregon State University
+CGVEH-GLDRCE-00917,,Node,1,Coastal Glider 917,COASTAL GLIDER,Teledyne Webb,GLD-0086_S,917,,20210403,,,Oregon State University
```
I have removed the 200M because all of the Gliders (at least the EA Coastal glider specifically) are considered interchangeable now and what may have started as a 200M glider may be deployed next as a 1000M glider (even have 350 M pumps now).  The configuration is captured per deployment via the water depth in each deployment csv file.
I plan in a future update to AM to remove all references of 200M or 1000M so that it does not become ambiguous in metadata per deployment.

rtravis-whoi replied:
Ah ok - sounds good.

snwhite replied:
Looks fine.

s-pearce commented:
@snwhite I removed you as a reviewer because I accidentally added you in the first place.  Rebecca has already reviewed.  Sorry and Thanks.

s-pearce commented:
> 
> 
> @snwhite I removed you as a reviewer because I accidentally added you in the first place. Rebecca has already reviewed. Sorry and Thanks.

@snwhite Whoops, reading closer I see that Collin added you as a reviewer.  Now doubly sorry about that. Adding you back in.

rtravis-whoi reviewed (commented):
Added ADCPA folder to Alfresco so if you post the ADCPAM file that is the last one that I need to confirm.  Thanks!

snwhite commented on bulk/eng_bulk_load-AssetRecord.csv:
```diff
@@ -1,544 +1,545 @@
-ASSET_UID,LEGACY_ASSET_UID,TYPE,Mobile,DESCRIPTION OF EQUIPMENT,MIO_Inventory_Description,Manufacturer,Model,Manufacturer's Serial No./Other Identifier,Firmware Version,ACQUISITION DATE,ORIGINAL COST,comments,MIO
-CGCON-AUVPNR-06263,,Sensor,1,Mobile Asset Controller for Coastal AUV,A6263 Controller,Hydroid,AUV,6263,,,,,Woods Hole Oceanographic Institution
-CGCON-AUVPNR-06264,,Sensor,1,Mobile Asset Controller for Coastal AUV,A6264 Controller,Hydroid,AUV,6264,,,,,Woods Hole Oceanographic Institution
-CGCON-BCPM01-00001,OL000483,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,0001,,,,CP01CNSM-00001-CPM1,Woods Hole Oceanographic Institution
```
Your update removed all of the leading zeros from our S/Ns.  Did you mean to do that?  Or did you do this in Excel and it just decided that leading zeros are not important?

rtravis-whoi replied:
OH wow good catch.  The whole csv is highlighted for me instead of just specific changes.  Didn't catch that.  And THAT'S why Collin wanted you to check these too!

s-pearce replied:
Sorry that was Excel being stupid with it's assumptions.

snwhite reviewed (commented):
I just reviewed the cross-mio stuff.  The big thing I noticed is all of our WHOI controller S/Ns got their leading zeros stripped off.  I don't think this was intentional and was probably due to using Excel or something to edit this file.

s-pearce commented:
@snwhite, @rtravis-whoi So I recently got a new computer with Windows 10 and Office 365 version of Excel.  Unlike previous versions of Excel when opening csv files, it definitely tampers with line endings in addition to removing the leading zeros for numeric only columns.  I don't see any advanced options that would control this behavior, so I will just have to do AM updates outside of AM from now on.  I am working on fixing these and will have an updated commit pushed hopefully tomorrow.

Additionally, the other `bulk_load` files all have Unix line endings, but `sensor_bulk_load` has Windows line endings, presumably changed in a previous pull-request.  In my fix should I convert `sensor_bulk_load` to have Unix line endings like the others?

s-pearce commented:
@rtravis-whoi & @snwhite Okay, I fixed the removal of leading zeros in `eng_bulk_load` and changed the line endings back to Unix style for the `eng_bulk_load`, `node_bulk_load`, and `eng_vocab`.  I also went ahead and changed the line endings on `sensor_bulk_load` to be Unix since all other files in the bulk directory had Unix line endings; I assume that it likely had Unix line endings in past commits too.

Also @rtravis-whoi I now have added the [ADCPAM](https://alfresco.oceanobservatories.org/alfresco/d/d/workspace/SpacesStore/9af09d91-261f-4854-b868-b382ee0933a0/ADCPA-M_DVL_SN_673246_Configuration_2021-03-04.pdf) file, which I named ADCPA-M_DVL_SN_673246_**Configuration**_2021-03-04.pdf with Configuration in the text since it isn't technically a calibration and TRDI titles it as Configuration and Test Report.  This can change if there is another standard filename already in place.

rtravis-whoi commented on calibration/ADCPAM/CGINS-ADCPAM-73246__20210304.csv:
```diff
@@ -0,0 +1,5 @@
+673246,CC_scale_factor1,0.61,
+673246,CC_scale_factor2,0.61,
+673246,CC_scale_factor3,0.61,
+673246,CC_scale_factor4,0.61,
```
looks good

rtravis-whoi commented on bulk/eng_bulk_load-AssetRecord.csv:
```diff
@@ -163,7 +163,8 @@ CGCON-GLDRCE-00382,OL000368,Sensor,1,Mobile Asset Controller for Coastal Glider,
 CGCON-GLDRCE-00386,OL000371,Sensor,1,Mobile Asset Controller for Coastal Glider,GL386 Controller,Teledyne Webb,G2 Slocum Gliders,386,,,,,Oregon State University
-CGCON-GLDRCE-00871,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL871 Controller,Teledyne Webb,G3 Slocum Gliders,871,,,,,Oregon State University
+CGCON-GLDRCE-00871,,Sensor,1,Mobile Asset Controller for Coastal Glider,G0871 Controller,Teledyne Webb,G3 Slocum Gliders,871,,,,,Oregon State University
+CGCON-GLDRCE-00917,,Sensor,1,Mobile Asset Controller for Coastal Glider,G0917 Controller,Teledyne Webb,G3 Slocum Gliders,917,,20210403,,,Oregon State University
```
updated file looks good

snwhite replied:
Looks good.

rtravis-whoi commented on bulk/sensor_bulk_load-AssetRecord.csv:
```diff
@@ -1,2568 +1,2573 @@
-CGINS-FLORTM-02819,N00118,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series M,FLORT for Coastal Slocum Glider,WET Labs,ECO Puck FLBBCD-SLK,2819,,20130701,186935,CGSN Glider Instrument Pool,
-CGINS-FLORTM-02858,N00016,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series M,FLORT for Coastal Slocum Glider,WET Labs,ECO Puck FLBBCD-SLK,2858,,20130125,,CGSN Glider Instrument Pool,
-CGINS-FLORTM-03015,,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series M,FLORT for Coastal Slocum Glider,WET Labs,ECO Puck FLBBCD-SLK,3015,,,,CGSN Glider Instrument Pool,Woods Hole Oceanographic Institution
-CGINS-FLORTM-03027,,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series M,FLORT for Coastal Slocum Glider,WET Labs,ECO Puck FLBBCD-SL
```
I had already reviewed the sensor additions; I think the whole file is now showing as changed because of Stuart changing the line endings to be Unix. So I think this is ok.

snwhite replied:
Additions and removal of references to OOI Glider Pool look good.

rtravis-whoi reviewed (approved):
The updates look good.

snwhite commented on bulk/eng_vocab.csv:
```diff
@@ -4,9 +4,9 @@ Coastal Wire-Following Profiler Controller,McLane,MMP-7
-Mobile Asset Controller for Open Ocean Glider,Teledyne Webb,G2 Slocum Gliders
+Mobile Asset Controller for Coastal Glider,Teledyne Webb,G2 or G3 Slocum Gliders
+Mobile Asset Controller for Profiling Glider,Teledyne Webb,G2 or G3 Slocum Gliders
+Mobile Asset Controller for Open Ocean Glider,Teledyne Webb,G2 or G3 Slocum Gliders
```
Looks good.

snwhite commented on bulk/node_bulk_load-AssetRecord.csv:
```diff
@@ -92,6 +92,7 @@ CGVEH-GLDRCE-00383,A00471,Node,1,Coastal Glider 383,COASTAL GLIDER 200M,Teledyne
 CGVEH-GLDRCE-00384,A00801,Node,1,Coastal Glider 384,COASTAL GLIDER 1000M,Teledyne Webb,CSG-1000,384,,20110628,186935,,Oregon State University
 CGVEH-GLDRCE-00386,A00473,Node,1,Coastal Glider 386,COASTAL GLIDER 1000M,Teledyne Webb,DG-1000,386,,20131011,186935,,Oregon State University
 CGVEH-GLDRCE-00871,,Node,1,Coastal Glider 871,COASTAL GLIDER 1000M,Teledyne Webb,GLD-0075_D,871,,20200813,,,Oregon State University
+CGVEH-GLDRCE-00917,,Node,1,Coastal Glider 917,COASTAL GLIDER,Teledyne Webb,GLD-0086_S,917,,20210403,,,Oregon State University
```
Looks good.

snwhite reviewed (approved):
I approved the 5 cross-mio files and those looked fine.

s-pearce commented:
Great.  Thank you both for your time for the review and catching those file mess ups.
As I said before I will avoid Excel for all future updates unless I can figure out how to make it not force changes.
Merging now.
